src/test/java/demo/multiagent/application/ActivityViewTest.java

```diff
@@ -0,0 +1,161 @@
+package demo.multiagent.application;
+
+import akka.javasdk.testkit.TestKit;
+import akka.javasdk.testkit.TestKitSupport;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import demo.multiagent.application.ActivityView.ActivityEntries;
+import demo.multiagent.application.ActivityView.ActivityEntry;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ActivityViewTest extends TestKitSupport {
+
+  @Override
+  protected TestKit.Settings testKitSettings() {
+    return TestKit.Settings.DEFAULT
+        .withAdditionalConfig(ConfigFactory.parseString("akka.javasdk.agent.openai.api-key=fake-key"))
```

Comment:
I changed my mind. 

Adding the fake key here to keep the CI file unchanged and in synch with the others.

I will apply this change in the sample in the akka-sdk repo